pkg/conduit/config.go

```diff
@@ -110,9 +110,9 @@ func DefaultConfig() Config {
 	cfg.Connectors.Path = "./connectors"
 	cfg.Processors.Path = "./processors"
 	cfg.Pipelines.Path = "./pipelines"
-	cfg.Schema.Type = SchemaTypeInMemory
+	cfg.SchemaRegistry.Type = SchemaRegistryTypeBuiltin
 
-	cfg.PluginDispenserFactories = builtin.DefaultDispenserFactories
+	cfg.ConnectorPlugins = builtin.DefaultBuiltinConnectors
```

Comment:
Nitpick: maybe we rename `DefaultBuiltinConnector` to just `DefaultConnectors` (it's in the `builtin` package anyway)?

Reply:
Idk about this, I get that we have `builtin` in the package name, but somehow `DefaultConnectors` feels odd without "builtin" in there. What is a "default connector"? 🤔

Reply:
To me it still reads nice: `DefaultConnectors` are the connectors you get by default. But as I said, just a nitpick, `builtin.DefaultBuiltinConnectors` is a good name as well!